src/app/Fake.IO.FileSystem/FileSystemInfo.fs

```diff
@@ -25,3 +25,9 @@ module FileSystemInfo =
         | :? FileInfo as file -> File(file)
         | :? DirectoryInfo as dir -> Directory(dir, dir.EnumerateFileSystemInfos())
         | _ -> failwith "No file or directory given."
+
+    let moveTo (fileSysInfo: FileSystemInfo) dest =
```

Comment:
We should either have docs or mark this `internal` (which is what I probably would go with for now).
If we want to add this public API we should briefly discuss the order of the arguments.

What I mean: We should decide if we expect this API to be used like `FileSystemInfo.moveTo src dst` or like

```fsharp
src
|> FileSystemInfo.moveTo dst
```
in which case we have to re-order the arguments.